bootstrap.php

```diff
@@ -1,20 +1,14 @@
 <?php
 
-declare(strict_types=1);
-
-require __DIR__ . '/../vendor/autoload.php';
+require __DIR__ . '/vendor/autoload.php';
 
 $configurator = new Nette\Configurator;
 
 //$configurator->setDebugMode('23.75.345.200'); // enable for your remote IP
-$configurator->enableTracy(__DIR__ . '/../log');
+$configurator->enableTracy(__DIR__ . '/log');
 
 $configurator->setTimeZone('Europe/Prague');
-$configurator->setTempDirectory(__DIR__ . '/../temp');
-
-$configurator->createRobotLoader()
-	->addDirectory(__DIR__)
-	->register();
```

Comment:
You do not need it then in composer.json

I usually find practical to have one folder indexed by robot loader which simplifies prototyping of potential libraries. I'm not sure if we want to have this in sandbox by default.